src/main/java/org/fenixedu/academic/domain/evaluation/config/MarkSheetSettings.java

```diff
@@ -62,26 +54,29 @@ static public boolean isRequiredNumberOfShifts(final int input) {
 
         if (!isRequiredAtLeastOneShift() && getInstance().getRequiredNumberOfShifts() != input) {
             throw new AcademicExtensionsDomainException("error.CompetenceCourseMarkSheet.shifts.required",
-                    String.valueOf(MarkSheetSettings.getInstance().getRequiredNumberOfShifts()));
+                    String.valueOf(getInstance().getRequiredNumberOfShifts()));
```

Comment:
If this method is no longer static, you're already calling over an instance of an object. If you call again `getInstance()`, you can be checking against another instance.